subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -184,14 +191,159 @@ static int parse_recv_state(const void *data, uint16_t length,
 	return 0;
 }
 
+static void bap_ntf_read_done(void)
+{
+	broadcast_assistant.busy = false;
+	net_buf_simple_reset(&cp_buf);
+}
+
+static uint8_t parse_and_send_recv_state(struct bt_conn *conn, const void *data,
+					 uint16_t length,
+					 struct bt_bap_scan_delegator_recv_state *recv_state)
+{
+	int err;
+	int16_t index;
+
+	err = parse_recv_state(data, length, recv_state);
+
+	if (err != 0) {
```

Comment:
```suggestion
	err = parse_recv_state(data, length, recv_state);
	if (err != 0) {
```

We genrally do not have empty lines between a value assignment and a verification check